lib/openstack/openstack_handle/planning_delegate.rb

```diff
@@ -0,0 +1,10 @@
+module OpenstackHandle
+  class PlanningDelegate < DelegateClass(Fog::Openstack::Planning)
```

Comment:
Typically, we've only created these Delegates for things that need to add some special functionality to the base service.  You can probably skip that here.

Rip this out and try without it.